central/main.go

```diff
@@ -777,8 +777,13 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 	)
 
 	if env.VulnReportingEnhancements.BooleanSetting() {
-		// Report custom routes
-		customRoutes = append(customRoutes, reportHandler.CustomRoutes()...)
+		// Append report custom routes
+		customRoutes = append(customRoutes, routes.CustomRoute{
+			Route:         "/api/reports/jobs/download",
```

Comment:
Did you choose to drop `{id}` (`/api/reports/jobs/{id}/download`) from the URL path because the custom route struct does not handle it correctly?

Reply:
Yes.